nixos/modules/services/misc/ethminer.nix

```diff
@@ -0,0 +1,114 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.ethminer;
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.ethminer = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Enable ethminer ether mining.";
+      };
+
+      recheckInterval = mkOption {
+        type = types.int;
+        default = 2000;
+        description = "Interval in milliseconds to farm recheck.";
```

Comment:
You still haven't changed this.. https://github.com/NixOS/nixpkgs/pull/55422#discussion_r272756608

Reply:
> I suggest "Interval in milliseconds between farm rechecks." or something like that instead

i do think this was changed as recommended?

Reply:
Ah yeah, sorry you did change it, but I had to mention it again because this sentence sounds wrong to me still, which is why I suggested "between" in the first place.

Reply:
agreed "between" makes it better. fixed.